.github/workflows/release.yaml

```diff
@@ -18,3 +18,9 @@ jobs:
         repository: absaoss/samlet
         dockerfile: Dockerfile
         tag_with_ref: true
+    - uses: dave-mcconnell/helm-gh-pages-microservices@master
```

Comment:
Maybe makes sense to also do some post-release smoke test, by deploying published chart from gh-pages to test kind cluster

Reply:
@somaritane done